datajoint_utilities/dj_search/lists.py

```diff
@@ -79,7 +79,7 @@ def drop_schemas(prefix, dry_run=True, ordered=False, force_drop=False):
             for schema_name in schemas_with_prefix:
                 try:
                     dj.schema(schema_name).drop(force=force_drop)
-                except OperationalError as e:
+                except (OperationalError, IntegrityError) as e:
```

Comment:
There is a slight logical error with this code. The exception saved in `recent_err` may not be the most important exception that causes the problem of not being able to drop the entire group of schemas. The informative error would be a foreign key constraint that references a table that is not included in the list, for example. It's fine for now. Perhaps listing all the errors would address the problem.